wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -2265,9 +2251,9 @@ fn test_create_dr_components_weighted_fee_weight_too_large() {
         witnesses: 1000,
         ..DataRequestOutput::default()
     };
-    let fee = 0;
+    let fee = Fee::default();
```

Comment:
This was a `FeeType::Weighted`

```suggestion
    let fee = Fee::relative_from_float(0.0);
```

Reply:
Oh I noticed this yesterday and fixed it but then it got lost during rebasing, apparently :cry: